drakrun/drakrun/config.py

```diff
@@ -41,9 +43,13 @@ def try_load() -> Optional['InstallInfo']:
         except FileNotFoundError:
             return None
 
+    @staticmethod
+    def delete() -> bool:
+        return safe_delete(InstallInfo._INSTALL_FILE_PATH)
+
```

Comment:
This function shouldn't return a bool and just crash if it fails to do its job (same as save).

Reply:
Sure, will update it